### PR TITLE
VEN-1110, VEN-1113, VEN-1114 | Customer Berth Views

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Add the user's id to the profile route
- Restructure the folders inside `profile`
- Implement missing components for the customer berths views
- Implement missing views and add hard-coded routes for testing 

### Closes :no_good_woman:
- [VEN-1110](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1110): Customer Berths page - Application
- [VEN-1113](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1113): Customer Berths page - Reoccurring Invoice
- [VEN-1114](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1114): Customer Berths page - Invoice paid

## Testing :alembic:
- From the profile page go the `Venepaikat` tab for the following users:
  - https://venepaikka.test.kuva.hel.ninja/fi/profile/1
  - https://venepaikka.test.kuva.hel.ninja/fi/profile/2
  - https://venepaikka.test.kuva.hel.ninja/fi/profile/3
  - https://venepaikka.test.kuva.hel.ninja/fi/profile/4
  - https://venepaikka.test.kuva.hel.ninja/fi/profile/5

## Screenshots :camera_flash:
image
image
image
image

### CODE REVIEW

File: src/features/profile/customerBerths/contractInfo/ContractInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  const contractStatus = `${t('page.profile.berths.contract.singed')} ${formatDate(signedAt, language)}`;
+
+  return (
+    <div className={classNames('vene-contract-info', className)}>
+      <h3 className="vene-contract-info__heading">{t('page.profile.berths.contract.label')}</h3>
+      <LabelValuePair label={t('page.profile.berths.contract.issued')} value={formatDate(issuedAt, language)} />
+      <LabelValuePair label={t('page.profile.berths.contract.edited')} value={formatDate(editedAt, language)} />
+      <LabelValuePair label={t('page.profile.berths.contract.status')} value={contractStatus} />
+      {/* FIXME */}
```
Comment (TatuArvela) [inline src/features/profile/customerBerths/contractInfo/ContractInfo.tsx:30]: Found a FIXME

File: src/locales/fi.json [hunk 2]
```diff
@@ -752,6 +753,24 @@
...
+          "label": "Sopimus",
+          "issued": "Luotu",
+          "edited": "Muokattu",
+          "singed": "Allekirjoitettu",
+          "status": "Tila",
+          "show": "Näytä sopimus"
+        },
+        "history": {
+          "label": "Venepaikkahistoria",
+          "berth": "paikka"
```
Comment (TatuArvela) [inline src/locales/fi.json:773]: Check that this is intended to be in lowercase

**Review by TatuArvela — APPROVED** [review]
Great job! 🥇 
I found two things to note, otherwise 100% 💎

File: src/locales/fi.json (hunk 2 above)
Comment (Nizar-Rahme) [reply src/locales/fi.json:773]: Yeah, that's intentional

File: src/features/profile/customerBerths/contractInfo/ContractInfo.tsx (hunk 1 above)
Comment (Nizar-Rahme) [reply src/features/profile/customerBerths/contractInfo/ContractInfo.tsx]: Fixed 😁

**Pull request merged**